src/main/java/core/basesyntax/QuadrangularArea.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface QuadrangularArea {
+    default double getQuadrangularArea(double firstSide, double secondSide) {
+        return firstSide * secondSide;
+    }
+}
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. The `getQuadrangularArea` method seems to be a generic operation that could apply to different quadrangular figures. However, if this operation is not common for all quadrilaterals in your domain, consider separating concerns into more specific interfaces or classes.